openstates/tx/votes.py

```diff
@@ -315,58 +315,41 @@ def scrape(self, chamber, session):
         if len(session) == 2:
             session = "%sR" % session
 
-        #As of 1/30/15, the 84th session does not have journals on the ftp
```

Comment:
Checked 6/3/17 and there still are not 84th session journals, nor 85th (current), so it doesn't seem to be a supported resource anymore. I removed the commented-out code.